### PR TITLE
Add system warnings, delivery notifications, general unread messages

### LINKED ISSUE #5018: Implement display of message statuses
### User Story

As a user, I want to see whether my message is Sent, Seen or not sent and resend if it attempt was unsuccessful.
### Description

*Type*: Feature

*Summary*: 
* in 1-1 Chat : 
1) Sent - [design](https://zpl.io/Vkml10Z)
2) Seen   
3) Not sent, Tap for options - [design](https://zpl.io/VqEOzY4)
* in Public Chat:
1) Sent - [design](https://zpl.io/Vkml10Z)
2) Not sent, Tap for options - [design](https://zpl.io/VqEOzY4)

### PR DESCRIPTION
**Pull request by vitvly**
Fixes #5018 
Duplicate of #5241 created in order to circumvent Jenkins build issues (branch base changed from `desktop` to `develop`).

### CODE REVIEW

**Review by maxhora — APPROVED** [review]

**Review by vkjr — APPROVED** [review]

File: src/status_im/ui/components/connectivity/view.cljs [hunk 1]
```diff
@@ -3,6 +3,7 @@
   (:require [re-frame.core :as re-frame]
             [reagent.core :as reagent]
             [status-im.ui.components.react :as react]
+            [taoensso.timbre :as log]
```
Comment (cammellos) [inline src/status_im/ui/components/connectivity/view.cljs]: unused import?

**Review by cammellos — APPROVED** [review]

**Comment by churik** [comment]
#5200 is fixed.

#5018 - for all messages can see "Not sent, tap for options" state
upd
Log: 
[StatusIm.txt](https://github.com/status-im/status-react/files/2241221/StatusIm.txt)

**Comment by churik** [comment]
@siphiuel 

1) No alignment for message statuses
status

2) If you click on "Not sent. Tap for options" status nothing happens. Two options should be available - Resend and Delete message

3) If you've got any system message ("Fetching message" for example) when you are somewhere in the middle of the history - you automatically are scrolled down to recent messages when system message disappears.

**Comment by vitvly** [comment]
Fixed items 1 and 3

**Comment by vitvly** [comment]
Item 2 - blurry warning icon must be due to https://github.com/status-im/react-native-desktop/issues/322. Tried using `react-native-popup-menu` to implement re-send options, but there are some issues (https://github.com/status-im/react-native-desktop/issues/321). So for now changed the label to simply `Not sent`.

**Comment by churik** [comment]
@siphiuel  thanks for fixes!
1 is fixed; 3 is fixed - but two more issues are appeared.

**4. now system warning is not visible due to overlapping with chat.**
overlap

**5. Sent status is shown for Not sent messages**
*Steps:*
1) install status-desktop and log in
2) turn off wi-fi
3) send message

*Expected result: * message is not sent; status is "Not sent"
*Actual result: * message  is not sent; status is "Sent"
Video is [here](http://take.ms/9uFqT)
Log: 
[StatusIm.txt](https://github.com/status-im/status-react/files/2332078/StatusIm.txt)

**Comment by vitvly** [comment]
Fixed item 4 - `now system warning is not visible due to overlapping with chat`

**Comment by maxhora** [comment]
Commits in this PR rebased and force pushed to try to overcome Jenkins build failure.

**Comment by churik** [comment]
@siphiuel 
checked last [MacOS](https://status-im.ams3.digitaloceanspaces.com/StatusIm.180830.081436.f7c3d7.pr.dmg) build and for me:
-  issue `5. Sent status is shown for Not sent messages` is still reproducible;
Video is  [here](http://take.ms/qI2Fq), Mac OS High Sierra (10.13.6 (17G65))

Tried on mailservers B, C, F - always can replicate.
App version: V0.0.1 (ad562deb3)
- and new behavior appears - when you send a message it jumps several times while fetching address.
Video is [here](http://take.ms/dQLGM)

**Comment by vitvly** [comment]
Item 5 must be related to https://github.com/status-im/react-native-desktop/issues/328

**Comment by churik** [comment]
Tested on desktop build (OS: Mac OSx High Sierra 10.13.6):

- installation
- account creating/restoring
- login/logout
- fetching 24 h history for public channels
- adding user to contact
- 1-1 chat
- mailservers switching
- offline mode
- messages statuses in 1-1 / public chats ('Not sent' is not implemented yet due to https://github.com/status-im/react-native-desktop/issues/328), separate issue will be created
- all issues above are fixed

Tested on IOS:
- installation
- account creating
- fetching 24 h history for public channels
- adding user to contact
- 1-1 chat
- messages statuses in 1-1 / public chats

**Comment by chadyj** [comment]
I ran into some issues with offline. Message was reported as sent.

**Pull request merged**